google/ggrc-core
# GGRC-4421 Add Effective and Last Deprecated dates to Section

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] #7252 (updates same tests)

# Issue description

Section lacks Effective Date and Last Deprecated Date which are used to track applicability of business objects.

# Steps to test the changes

Export a Section. It should contain Effective Date and Last Deprecated Date fields.

- Effective Date should be importable
- Last Deprecated Date should not be importable
- Last Deprecated Date should be set to current date once Section is moved to Deprecated state

# Solution description

- Timeboxed mixin is split in two to enable adding just Effective, not Stop date
- Section now mixes WithLastDeprecatedDate and WithEffectiveDate in

# Sanity checklist




# PR Review checklist
